### DISCUSSION TITLE
No trace in UI

### DISCUSSION THREAD

**Discussion by sruby** [apache/skywalking#8873] [Q&A] [answered]
Yesterday I installed skywalking 8.9.1.
Yesterday, the agent sent segment normally, the dashboard showed service and instance normally, but no trace information.
This morning, I found that the UI was displaying yesterday's trace information normally, but not today's trace information.
OAP, ES have no error log.
ES、OAP、Application time is same, all set  env TZ=Asia/Shanghai.
What should I do to troubleshoot the problem?

Thanks!

**Comment by wu-sheng** [comment]
Hard to follow a question like this.😅

**Reply by sruby** [reply]
When checking the browser request parameters, I found that the time interval is missing a few hours.
This is caused by ignoring the browser's time zone, not noticing that the browser's time zone is UTC +0, not UTC +8.

**Reply by Fine0830** [reply]
It seems that the duration range is cached in localStorage. You need to refresh pages with the icon on the page header.

**Comment by sruby** [comment, marked as answer]
Configure the correct browser time zone and it's solved.
